ui/libproject/devstart.rb

```diff
@@ -54,16 +48,10 @@ def dev_up(*args)
 
   options = DevUpOptions.new.parse(args)
 
-  ensure_git_hooks
-
   unless Dir.exist?("node_modules")
     install_dependencies
   end
 
-  unless File.exist?(SWAGGER_CODEGEN_CLI_JAR)
-    download_swagger_codegen_cli
-  end
-
   ENV["ENV_FLAG"] = options.env == "local" ? "" : "--environment=#{options.env}"
   common.run_inline %W{docker-compose up -d}
   at_exit { common.run_inline %W{docker-compose down} }
```

Comment:
`clean_git_hooks` (lower down) is obsolete. Might want to chop it as part of this update.

Reply:
Trimmed.